Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -1378,6 +1519,18 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>1.4</string>
+			<key>pfm_description</key>
+			<string>If defined it specifies the custom location of the appication to be given access to the keychain item.</string>
```

Comment:
Small typo - `appication` >> `application`

Reply:
The typo appears to be in the documentation as well: https://gitlab.com/orchardandgrove-oss/NoMADLogin-AD/-/blob/develop/Base/Preferences.swift